cmd/updater/diffdumps/cmd.go

```diff
@@ -98,12 +148,18 @@ func generateK8sDiffs(outputDir string, baseZipR *zip.ReadCloser, headZipR *zip.
 			continue
 		}
 
-		// Only look at YAML files in the k8s/ folder.
-		if filepath.Dir(name) != vulndump.K8sDirName || filepath.Ext(name) != ".yaml" {
+		// Only look at YAML files in the k8s/ or istio/ folder.
+		if filepath.Dir(name) != dirName || filepath.Ext(name) != ".yaml" {
 			continue
 		}
-		if err := generateK8sDiff(k8sSubDir, baseFiles[name], headF); err != nil {
-			return errors.Wrapf(err, "generating Kubernetes diff for file %q", headF.Name)
+		if outputDir == vulndump.K8sDirName {
```

Comment:
Instead of doing this check, maybe we pass in a `generateDiffFunc` to this function since they both have the same function signature